source/base/Timer.ooc

```diff
@@ -40,34 +40,28 @@ Timer: abstract class {
 		this _average = this _total / this _count
 	}
 	reset: virtual func {
-		this _startTime = 0
-		this _endTime = 0
-		this _result = 0
-		this _total = 0
-		this _count = 0
-		this _min = 0
-		this _average = 0
+		(this _min, this _max, this _startTime, this _endTime, this _result, this _total, this _count, this _average) = (Double positiveInfinity, 0, 0, 0, 0, 0, 0, 0)
```

Comment:
I put the nonzero assignemtn on its own line. Maybe she won't complain too much now. :)